DfE.FindInformationAcademiesTrusts/assets/sass/components/_export-button.scss

```diff
@@ -0,0 +1,14 @@
+.export-button--with-icon {
+    display: flex;
+    align-items: center;
+    padding: 10px 15px;
+    font-size: 16px;
+    background-color: #f8f8f8;
+    border: 1px solid #dcdcdc;
+    color: #0b0c0c;
+}
+
+.export-button__icon {
+    margin-right: 8px;
+    fill: currentColor;
```

Comment:
I've seen the issue raised by the linter. There is some discussion here https://github.com/stylelint/stylelint/issues/5863 about which version is correct/acceptable. We can choose to deal with it either by:

- Switching this line to `fill: currentColor;`
- Adding `/* stylelint-disable-next-line value-keyword-case */`
- Enabling camelCaseSvgKeywords as seen in the link I included

Reply:
Yes it does, I can confirm again in DevTest to make sure my memory isn't foggy!

Reply:
After testing it did when I changed it on the SVG but not the component file, so I will update this file to consistent styling, good spot 👍